cmd/kind/completion/completion.go

```diff
@@ -46,6 +47,8 @@ This depends on the bash-completion binary.  Example installation instructions:
 # for zsh users
 	% kind completion zsh > /usr/local/share/zsh/site-functions/_kind
 	% autoload -U compinit && compinit
+  or if zsh-completion is installed via homebrew
+    % kind completion zsh > "${fpath[1]}/_kind"
```

Comment:
please change to:
```
# or if zsh-completion is installed via homebrew
	% kind completion zsh > "${fpath[1]}/_kind"
```
note that tag before `%`.

Reply:
Changed. Thanks.